src/test/java/com/artipie/VertxMainITCase.java

```diff
@@ -27,7 +26,6 @@
  * @since 0.1
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
-@Disabled("FIXME: this test is failing on JVM-17, checks the logs")
```

Comment:
We also need to move this test to testcontainers (like other working tests here) as described in todo.

Reply:
Ok

Reply:
Good now :)